debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -609,7 +609,7 @@ public static GtidNewChannelPosition parse(String value, String defaultValue) {
                     "Password to unlock the keystore file (store password) specified by 'ssl.trustore' configuration property or the 'javax.net.ssl.trustStore' system or JVM property.");
 
     public static final Field TABLES_IGNORE_BUILTIN = RelationalDatabaseConnectorConfig.TABLE_IGNORE_BUILTIN
-            .withDependents(DATABASE_INCLUDE_LIST_NAME, DATABASE_WHITELIST_NAME);
```

Comment:
Did that cause issues?

Reply:
Yes and no. I tried to cover that in the PR description on DBZ-2527. It will create a validation warning about that the dependent is not defined. That makes imo sense for the new include/exclude lists but should not show a warning for legacy whitelist/blacklist fields. That's why I removed the legacy fields.

Reply:
Gotcha, that makes sense. Thanks for clarifying!